### DISCUSSION TITLE
Strange Connection Failure after initial connection success

### DISCUSSION THREAD

**Discussion by gopuma** [awslabs/aws-mysql-jdbc#555] [Q&A]
Aurora MySQL with 2 replica and 1 master.
RO instances(replicas) uses custom domain with AWS route53 domain CNAME(RW uses default end point)
ClusterInstanceHostPattern parameter set.
Client trying to connect with two different databases with Route53 custom domain(xxx.xxx.com, yyy.yyy.com)
As soon as the client application starts up the connection succeeded without errors. When POST request or any other subsequent request is made the connection failed. Two domains are queried successfully in the client application with dig tool. When client application connects to Aurora MySQL with reader end point the problem goes away.

**Comment by jasonli-improving** [comment]
Hi @gopuma,

Thanks for reaching out and raising this issue.

We'll take a look at this and keep you updated as we investigate.

In the meantime, could you provide some additional details regarding the following?

- Could you provide driver logs containing the error?
- Are you able to provide any sample code or an example of your connection string?
- What version of the driver are you using?
- Are there any libraries or frameworks that you are using?

To enable driver logging, please refer to our documentation [here](https://github.com/awslabs/aws-mysql-jdbc/blob/main/README.md#enable-logging).

Thank you for your patience!
